proxy/src/console/provider/neon.rs

```diff
@@ -164,46 +169,56 @@ impl super::Api for Api {
         &self,
         extra: &ConsoleReqExtra,
         creds: &ComputeUserInfo,
-    ) -> Result<CachedRoleSecret, GetAuthInfoError> {
-        let ep = creds.endpoint.clone();
-        let user = creds.inner.user.clone();
-        if let Some(role_secret) = self.caches.role_secret.get(&(ep.clone(), user.clone())) {
-            return Ok(role_secret);
+    ) -> Result<Option<CachedRoleSecret>, GetAuthInfoError> {
```

Comment:
It's not clear to me why this is now optional

Reply:
Added comment with explanation to the trait definition (provider.rs).

Reply:
Clear, thank you